### ISSUE TITLE
Could you give the simple example in 1D case？

### ISSUE THREAD

**Issue by damonwan1** [rjbruin/flexconv#6]
Cause I found in the experiments directory ：
![image](https://github.com/rjbruin/flexconv/assets/33806362/fabfb594-534f-4ed3-b043-0414a0a0b401)

I wonder how to use flexconv to solve the problem：
1. I have total 20000  nums 1D time series data with channel 21. For example, standard Weather dataset with 20 argument to predict OT.
2.  I have different sequence length, such as [21,4] , [21,6] , [21, 11] . [21, 2]....
3. I WANT to use the flexconv to finish the work that transforming all the sequence to [21, 8].

Does this case is suitable for flexconv ?
How can I train the flexconv and finish the work above??

Hope for your response!!
Thank you very much for the good job!

**Comment by rjbruin** [comment]
@dwromero How do we deal with varying sequence lengths in 1D? Padding, I presume?

**Comment by dwromero** [comment]
Yes, although it is not necessary, it is practical to do it this way to vectorize operations across all input samples :) 

(sorry for the long wait, I do not have access to d.w.romeroguzman@vu.nl anymore).
